src/decls/PropTypes.js

```diff
@@ -6,8 +6,16 @@ const StampyPropTypes = {
 
     className: PropTypes.string,
 
+    element: PropTypes.oneOfType([
+        PropTypes.element,
+        PropTypes.string,
+        PropTypes.func
+    ]),
+
     htmlProps: PropTypes.object,
 
+    modifier: PropTypes.string,
```

Comment:
Is this one necessary / being used? I think there's already a `StampyPropTypes.spruceModifier`, although the naming's a little inconsistent with the `modifier` prop name.